templates/jinja2/index.html

```diff
@@ -16,21 +16,7 @@
 <div class="row mb-5">
   <div class="col-md-10 offset-md-1 pl-md-0">
     <div class="alert alert-warning" role="alert">
-      <p class="lead">
-        <strong>Dear BGA readers,</strong>
-      </p>
-
-      <p>
-        First, thanks very much for visiting our Salary Database site. We know hundreds of thousands of people use it throughout the year and find it useful.
-      </p>
-
-      <p>
-        <strong>But we need your help to keep this important source of information going.</strong> This database costs money and time to complete. We contract with outside organizations, and BGA staffers spend more than a year requesting, compiling, organizing and checking data from hundreds of government bodies across Illinois to bring this site to you for free. We don't run ads and we are a small nonprofit.
-      </p>
-
-      <p>
-        If every person who visited this site gave us $1, we could complete our fundraising for the year right now. If you find this database valuable, please take just a few minutes to support it. Thank you.
-      </p>
+      {{ donate_message|safe }}
 
       <p class="text-center">
         <a href="https://donate.bettergov.org/?utm_source=database&utm_medium=other" class="btn btn-primary btn-lg" target="_blank">
```

Comment:
same with the button, too i think @smcalilly

Reply:
@fgregg changed here: 6ad320a